### DISCUSSION TITLE
Keeps crashing after reboot;

### DISCUSSION THREAD

**Discussion by Pataplu** [Geektoolkit/Dynaframe3#94] [Q&A]
Please help me figure this out. When the raspberry starts the screen turns black and after a minute or so it returns to the raspberry home screen.
When I run Dynaframe from the command line I get this output:

01/03/2021_ 08:16:Tried to fix rendering
01/03/2021 08:16:Rotating to Portrait. calculated  W: 1080 H: 1920
01/03/2021 08:16:Rotation to Portrait. Panel  Width: 1920 Height: 1080
01/03/2021 08:16:Rotation to Portrait. Window Width: 1920 Height: 1080
01/03/2021 08:16:Reload settings was true
01/03/2021 08:16:Refresh Directories was true
01/03/2021 08:16:ERROR: GoToNext was unable to find current item in playlist!
01/03/2021 08:16:ERROR: Selected Playlist Folder Missing: /home/pi/Pictures/F1
01/03/2021 08:16:New List generated! Contains: 0 items.
01/03/2021 08:16:----------------------Begin Playlist Dump----------------
01/03/2021 08:16:------------------- END PLAYLIST DUMP-------------------
01/03/2021 08:16:New List generated! Contains: 5 items.
01/03/2021 08:16:----------------------Begin Playlist Dump----------------
01/03/2021 08:16:Image:/home/pi/Pictures/JackBrabham.jpg
01/03/2021 08:16:Image:/home/pi/Pictures/MikeHawthorn.jpg
01/03/2021 08:16:Image:/home/pi/Pictures/GiusepeFarina.jpg
01/03/2021 08:16:Image:/home/pi/Pictures/AlbertoAscari.jpg
01/03/2021 08:16:Image:/home/pi/Pictures/JuanManueFangio.jpg
01/03/2021 08:16:------------------- END PLAYLIST DUMP-------------------
01/03/2021 08:16:Next file is: /home/pi/Pictures/AlbertoAscari.jpg
01/03/2021 08:16:PlayImageFile() called
omxplayer.bin: no process found
01/03/2021 08:16:Initialized
Unhandled exception. System.Net.HttpListenerException (98): Address already in use
   at System.Net.HttpEndPointManager.GetEPListener(String host, Int32 port, HttpListener listener, Boolean secure)
   at System.Net.HttpEndPointManager.AddPrefixInternal(String p, HttpListener listener)
   at System.Net.HttpEndPointManager.AddListener(HttpListener listener)
   at System.Net.HttpListener.Start()
   at SimpleHTTPServer.Listen(Object obj)
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart(Object obj)
Aborted__

**Comment by Geektoolkit** [comment]
A few things that could cause this:
1) Are you starting it via VNC? If so there is a bug where moving the mouse will crash the UI.  If that's the case please don't move the mouse after reboot just to make sure it's not that
2) Before running it from commandline please try "sudo pkill Dynaframe".  The old instance may be running

The exception you're seeing on the command line is saying that the port (8000) is in use by something.  So we have to figure out what that is.

Also please try an 'ifconfig' from the command line and then on another machine go to http://(your_ipaddress):8000 and let us know if you're seeing a user interface.  That'll let me know if the webserver is running and we have a UI issue, or if the whole thing is crashing

**Comment by Pataplu** [comment]
Tnx. I think it was the mouse thing because I started it up without mouse and keyboard attached and it worked.

br Patrick

**Reply by Geektoolkit** [reply]
Excellent ok, that's a known issue, we have some ideas on what's causing it and will work to try to figure out a fix. Basically the system that hides the mouse pointer causes it. Glad it's working when you dont' move the mouse for now!
